src/components/observation/render/index.js

```diff
@@ -10,7 +10,7 @@ export const RenderAlcohol = (props) => {
   return (
     <ObservationStatusPercentageComment
       disabled={props.disabled}
-      statusValue={props.valueProps.status}
+      statusValue={props.valueProps.statusValue}
```

Comment:
My previous question is highlighted here. It reads better without the Value suffix I think.